math/bigFloat_wrapper_test.go

```diff
@@ -0,0 +1,113 @@
+package math
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBigFloatSub_Panic(t *testing.T) {
+	value1 := new(big.Float)
+	_ = value1.GobDecode([]byte{1, 10, 0, 0, 0, 53, 0, 0, 0, 58, 0, 31, 28, 26, 150, 254, 14, 45})
+	value2 := new(big.Float)
+	_ = value2.GobDecode([]byte{1, 10, 0, 0, 0, 53, 0, 0, 0, 52, 222, 212, 49, 108, 64, 122, 107, 100})
+	result, err := SubBigFloat(value1, value2)
+
+	require.Equal(t, big.NewFloat(0).String(), result.String())
```

Comment:
as discussed, this can be replaced to  
```
require.Equal(t, big.NewFloat(0), result)
```

if the defer functions rewrite the result pointer